EIPS/7639.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 7639
```

Comment:
@lightclient need to rename the file to eip-7639.md

Reply:
fixed, thx